# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sleuthkit/missing-arflags.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- sleuthkit-2.06/src/auxtools/Makefile.orig	2006-10-09 20:56:57.000000000 +0200
+++ sleuthkit-2.06/src/auxtools/Makefile	2006-10-09 21:12:18.000000000 +0200
@@ -21,7 +21,7 @@
 manpages: 
 
 lib:	$(LIBOBJ)
-	$(AR) $(LIB) $?
+	$(AR) $(ARFLAGS) $(LIB) $?
 	$(RANLIB) $(LIB)
 
 indent:
--- sleuthkit-2.06/src/fstools/Makefile.orig	2006-10-09 21:13:59.000000000 +0200
+++ sleuthkit-2.06/src/fstools/Makefile	2006-10-09 21:19:07.000000000 +0200
@@ -32,7 +32,7 @@
 all:	lib $(PROGS)
 
 lib:	$(LIBOBJ)
-	$(AR) $(LIB) $?
+	$(AR) $(ARFLAGS) $(LIB) $?
 	$(RANLIB) $(LIB)
 
 $(BIN_DIR)/dls: dls.o $(LIB) 
--- sleuthkit-2.06/src/hashtools/Makefile.orig	2006-10-09 21:04:40.000000000 +0200
+++ sleuthkit-2.06/src/hashtools/Makefile	2006-10-09 21:08:39.000000000 +0200
@@ -16,7 +16,7 @@
 all:    lib $(PROGS)
 
 lib: $(LIBOBJ)
-	$(AR) $(LIB) $?
+	$(AR) $(ARFLAGS) $(LIB) $?
 	$(RANLIB) $(LIB)
 
 $(BIN)/md5: md5.o md5c.o
--- sleuthkit-2.06/src/imgtools/Makefile.orig	2006-10-09 21:02:08.000000000 +0200
+++ sleuthkit-2.06/src/imgtools/Makefile	2006-10-09 21:14:58.000000000 +0200
@@ -19,7 +19,7 @@
 all:	lib $(PROGS)
 
 lib:	$(LIBOBJ)
-	$(AR) $(LIB) $?
+	$(AR) $(ARFLAGS) $(LIB) $?
 	$(RANLIB) $(LIB) 
 
 $(BIN_DIR)/img_stat: img_stat.o $(LIB)
--- sleuthkit-2.06/src/mmtools/Makefile.orig	2006-10-09 21:15:26.000000000 +0200
+++ sleuthkit-2.06/src/mmtools/Makefile	2006-10-09 21:15:41.000000000 +0200
@@ -19,7 +19,7 @@
 all:	lib $(PROGS)
 
 lib:	$(LIBOBJ)
-	$(AR) $(LIB) $?
+	$(AR) $(ARFLAGS) $(LIB) $?
 	$(RANLIB) $(LIB)
 
 $(BIN_DIR)/mmls: mmls.o $(LIB) 
--- sleuthkit-2.06/src/srchtools/Makefile.orig	2006-10-09 21:16:16.000000000 +0200
+++ sleuthkit-2.06/src/srchtools/Makefile	2006-10-09 21:16:34.000000000 +0200
@@ -17,7 +17,7 @@
 all:    lib $(PROGS)
 
 lib: $(LIBOBJ)
-	$(AR) $(LIB) $?
+	$(AR) $(ARFLAGS) $(LIB) $?
 	$(RANLIB) $(LIB)
 
 $(BIN)/srch_strings: srch_strings.o $(LIB)
